dev/com.ibm.ws.kernel.boot.ws-server/publish/bin/server

```diff
@@ -1046,7 +1046,12 @@ javaCmd()
     # trap on the signals here to ensure the background tail process ends.
     trap "exit" INT TERM
     trap "killSilent $TAIL_PID" EXIT
-    "${JAVA_CMD}" "$@" >> "${CHECKPOINT_CONSOLE_LOG}" 2>&1 < /dev/null
+    #On some shells (bash) the shell will issue an ugly message to STDERR when criu kills the process.
+    # We direct STDERR to /dev/null to avoid that noise.
+    {
+      #  Launch with stdin set to /dev/null to avoid file-handle complications on criu restore
+      "${JAVA_CMD}" "$@" >> "${CHECKPOINT_CONSOLE_LOG}" 2>&1 < /dev/null
+    } 2> /dev/null
```

Comment:
We had discussed piping both 1 and 2 to /dev/null here

```
    } 2>&1 > /dev/null
```

Reply:
The reason for piping both stderr and stdout was that I had incorrectly stated that on ubuntu  it was emitting the message to stdout. But I was mistaken. It is emitting it to stderr as on RHEL. The reason the "Killed" is still appearing on ubuntu is that it seems to be emitted to the same stderr stream that the forked java process is writting to and appears in the `${CHECKPOINT_CONSOLE_LOG}`. Whereas on RHEL it seems to be emitted by the STDERR of bash itself and so does NOT appear in the `${CHECKPOINT_CONSOLE_LOG}`. So I'm unable to block the message on ubuntu without doing somting more aggressive like filtering. However since we are targetting container using RHEL-UBI this release, it seems a valid fix. Also the message as formatted by bash looks more like something went wrong than the one on ubuntu, which is just a single word, "Killed".

The distinction between behavior on RHEL/Ubuntu is probably down to the fact that  `#!/bin/sh` on RHEL invokes bash by default whereas on ubuntu it invokes dash which is supposed to be a POSIX compliant stand-in for /bin/sh